### DISCUSSION TITLE
how can i use police based ipsec in my side and route based ipsec on other side

### DISCUSSION THREAD

**Discussion by shangji2905** [strongswan/strongswan#1142] [General]
s-gw is using route based ipsec ,and our side only can use police based ipsec , is there any way we can establish a tunnel between us.

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.

**Describe the solution you'd like**

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions, workarounds or
features you've considered.

**Additional context**
Add any other context about the feature request here.

**Comment by tobiasbrunner** [comment]
Yes, it's basically a local decision to use a route-based or policy-based configuration (unless it's based on GRE).

**Reply by shangji2905** [reply]
iked_tunnel_event_add_event_in_sa_cfg: Updating tunnel-event for sa-cfg instance-EXTERNAL-PKI_67123991 with No nexthop-tunnel IP received from peer. Negotiation failed
 iked_tunnel_event_add_event_in_sa_cfg: tunnel-event No nexthop-tunnel IP received from peer. Negotiation failed count incremented to 46

this log is printed by s-gw, due to there is no nexthop-tunnel IP from our side's auth request so negotiation failed.

**Reply by tobiasbrunner** [reply]
No idea what that "nexthop-tunnel IP" could be, so you better ask the devs of that implementation what they are expecting.
